ajax/libs/vertx/3.0.0/vertx-eventbus.js

```diff
@@ -0,0 +1,228 @@
+/*
+ *   Copyright (c) 2011-2015 The original author or authors
+ *   ------------------------------------------------------
+ *   All rights reserved. This program and the accompanying materials
+ *   are made available under the terms of the Eclipse Public License v1.0
+ *   and Apache License v2.0 which accompanies this distribution.
+ *
+ *       The Eclipse Public License is available at
+ *       http://www.eclipse.org/legal/epl-v10.html
```

Comment:
I think @clairetsai818 removed it in package.json though I don't know why, seems to be added back now.